merkle/hstar2_test.go

```diff
@@ -130,20 +131,25 @@ func TestHStar2GetSet(t *testing.T) {
 // 256-prefixSize, and can be passed in as leaves to top-subtree calculation.
 func rootsForTrimmedKeys(t *testing.T, prefixSize int, lh []HStar2LeafHash) []HStar2LeafHash {
 	var ret []HStar2LeafHash
-	s := NewHStar2(treeID, maphasher.Default)
+	hasher := maphasher.Default
+	s := NewHStar2(treeID, hasher)
 	for i := range lh {
-		prefix := new(big.Int).Rsh(lh[i].Index, uint(s.hasher.BitLen()-prefixSize))
-		b := lh[i].Index.Bytes()
-		// ensure we've got any chopped of leading zero bytes
-		for len(b) < 32 {
-			b = append([]byte{0}, b...)
+		subtreeDepth := s.hasher.BitLen() - prefixSize
+		prefix := lh[i].Index.Bytes()
+		// ensure we've got any chopped off leading zero bytes
```

Comment:
I don't think that comment is fully clear. I know it was there before but how about "left pad prefix with any zero bytes that were previously removed".